tools/sync_test_server/ros-testing-server.js

```diff
@@ -51,7 +51,9 @@ function waitForRosToInitialize(attempts, onSuccess, onError, startSequence) {
     http.get("http://0.0.0.0:9080/health", function(res) {
         if (res.statusCode != 200) {
             winston.info("ROS /health/ returned: " + res.statusCode)
-            waitForRosToInitialize(attempts - 1, onSuccess, onError, startSequence)
+            setTimeout(function() {
+                waitForRosToInitialize(attempts - 1, onSuccess, onError, startSequence);
+            }, 500);
```

Comment:
Note that in reality, when ROS is not started you'll get the `ECONNREFUSED` error which is caught by the `.on('error')` block, which actually does the wait. This is just a double check when somehow the Webserver (Express) is up but the health endpoint is not successful (`200`)